### DISCUSSION TITLE
Support request: bug: shell commands result in 'The application panicked (crashed). Message:  called `Option::unwrap()` on a `None` value'

### DISCUSSION THREAD

**Discussion by bneqld** [aws/codewhisperer-command-line-discussions#87] [Support ticket] [answered]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

cw command line commands work (login, ai, doctor...)

### Actual behaviour

Error message:
cw login
The application panicked (crashed).
Message:  called `Option::unwrap()` on a `None` value
Location: /private/tmp/45f9383e-50e0-44bf-94f1-5d5fb2a2b3f0/lib/fig_util/src/shell.rs:91

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.

### Environment

```yaml
running cw diagnostic does not work resulting in the same error message.
I have already reainstalled and restarted cw, I also tried a new shell. My MacOS device is endpoint managed, and I can only install cw using an option 'Install with Priviledge Management' provided by the endpoint management solution before.
cw did work fully before. Problems started after the application self-updated.
```

**Comment by grant0417** [comment]
Hey, could you run `bash --version` and share the output? It seems like our bash version detection logic might have an issue.

**Comment by brendanfalk** [comment]
Hey @bneqld just checking in. Could you share your bash details so we can investigate?

**Comment by yuuitiro** [comment]
I also encountered the same phenomenon. The version of MacOS X is:

$ sw_vers
ProductName:            macOS
ProductVersion:         14.4.1
BuildVersion:           23E224

$ env LANG=C /bin/bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin23)
Copyright (C) 2007 Free Software Foundation, Inc.
The version of cw is as follows:

$ cw --version
cw 0.15.3

**Reply by grant0417** [reply, marked as answer]
Thanks, this will be resolved in the next version.

**Reply by grant0417** [reply]
The new version `1.0.1` was just released and no longer will panic here.

**Reply by yuuitiro** [reply]
👍  I will try 1.0.1 !
